src/components/Sections/WidgetEmptyState.less

```diff
@@ -0,0 +1,20 @@
+@import (reference) "../../css/variables";
+
+.widget-empty-state {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  flex-direction: column;
+  height: 100%;
+
```

Comment:
we should show the original size of the widget 
including the widget title 

![image](https://user-images.githubusercontent.com/61918543/115239085-0be0a400-a127-11eb-8029-b2a9a0c80134.png)